<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Game1Score</class>
 <widget class="QWidget" name="Game1Score">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: black</string>
  </property>
  <widget class="QTextBrowser" name="textValues">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>100</y>
     <width>256</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:white;</string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Asap Condensed'; font-size:16pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textVices">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>100</y>
     <width>256</width>
     <height>461</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
border:none;
color:white;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>LIVES:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>TIME LEFT:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>360</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>44</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>SCORE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>YOUR VALUES:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string>YOUR VICES:</string>
   </property>
  </widget>
  <widget class="QLabel" name="timeLeftValue">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>200</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="scoreValue">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>360</y>
     <width>161</width>
     <height>100</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>44</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color:white;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>1180</x>
     <y>500</y>
     <width>51</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Asap Condensed</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:focus { outline-color:white; 
border:1px solid #ffffff;}
QPushButton{
background:transparent;
border:none;
color:white;
}
</string>
   </property>
   <property name="text">
    <string>EXIT</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
